osu.Game/Beatmaps/Formats/LegacyDecoder.cs

```diff
@@ -168,23 +168,36 @@ public class LegacyDifficultyControlPoint : DifficultyControlPoint, IEquatable<L
             /// </summary>
             public double BpmMultiplier { get; private set; }
 
+            /// <summary>
+            /// Whether or not slider ticks should be generated at this control point.
+            /// This exists for backwards compatibility with maps that abuse NaN slider velocity behavior on osu!stable (e.g. /b/2628991).
+            /// </summary>
+            public bool GenerateTicks { get; private set; } = true;
```

Comment:
```suggestion
            public bool readonly GenerateTicks = true;
```

Should work now?

Reply:
That breaks `CopyFrom`

Reply:
I'm not sure that's ever used, might be a good idea to `NotImplementedException` it if we can.

Reply:
It's used in `ControlPoint.DeepClone`.
![image](https://user-images.githubusercontent.com/11239786/186342832-4fe8a36e-9069-48fa-9609-7541392513ad.png)